code/modules/paperwork/paperplane.dm

```diff
@@ -27,14 +27,6 @@
 		internalPaper = new /obj/item/weapon/paper(src)
 	update_icon()
 
-/obj/item/weapon/paperplane/suicide_act(mob/user)
```

Comment:
Definitely one of the stranger `suicide_act`'s I've come across. Although it get's creativity points for use of `adjust_eye_damage`.